notesapi/v1/views.py

```diff
@@ -19,31 +21,25 @@ class AnnotationSearchView(APIView):
     def get(self, *args, **kwargs):  # pylint: disable=unused-argument
         """
         Search annotations.
-
-        This method supports the limit and offset query parameters for paging
-        through results.
         """
         params = self.request.QUERY_PARAMS.dict()
-
-        if 'offset' in params:
-            kwargs['offset'] = _convert_to_int(params.pop('offset'))
-
-        if 'limit' in params and _convert_to_int(params['limit']) is not None:
-            kwargs['limit'] = _convert_to_int(params.pop('limit'))
-        elif 'limit' in params and _convert_to_int(params['limit']) is None:  # bad value
-            params.pop('limit')
-            kwargs['limit'] = settings.RESULTS_DEFAULT_SIZE
+        for field in ('text', 'quote'):
+            if field in params:
+                params[field + "__match"] = params[field]
+                del params[field]
+        if params.get('highlight'):
+            params.pop('highlight')
+            results = NoteMappingType.process_result(
+                list(
+                    note_searcher.query(**params).order_by("-created").values_dict("_source") \
+                        .highlight("text", pre_tags=['<span>'], post_tags=['</span>'])
```

Comment:
it does not, and with recent switch to `haystack` that is even more complicated. (haystack does not support it)